content/platforms/kubernetes/getting-started/quick-start.md

```diff
@@ -0,0 +1,228 @@
+can be found by checking the [releases on Github](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases)
+or via the GitHub API.
+
+The bundle for the latest release can be downloaded via `curl` using the following:
+
+```
+VERSION=`curl --silent https://api.github.com/repos/RedisLabs/redis-enterprise-k8s-docs/releases/latest | grep tag_name | awk -F'"' '{print $4}'`
+curl --silent -O https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/$VERSION/bundle.yaml
+```
+
+or you can replace `VERSION` in the above with a specific release tag you desire.
+
+
+If you inspect this YAML file, you will find definitions for the various items
+described in the previous section.
+
+You can install the operator with a single apply command:
+
+```
+kubectl apply -f bundle.yaml
+```
+
+You should see a result similar to:
+
+```
+role.rbac.authorization.k8s.io/redis-enterprise-operator created
+serviceaccount/redis-enterprise-operator created
+rolebinding.rbac.authorization.k8s.io/redis-enterprise-operator created
+customresourcedefinition.apiextensions.k8s.io/redisenterpriseclusters.app.redislabs.com configured
+customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redislabs.com configured
+deployment.apps/redis-enterprise-operator created
+```
+
+## Verifying the operator is running
+
+You can verify the operator is running in your namespace checking the deployment:
+
+```
+kubectl get deployment -l name=redis-enterprise-operator
+```
+
+You should see a result similar to:
+
+```
+NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
+redis-enterprise-operator   1/1     1            1           0m36s
+```
+
+## Testing the operator
+
+A cluster is created by creating a custom resource with the kind "RedisEnterpriseCluster"
+that contains the specification of the cluster option. See the
+[Redis Enterprise Cluster API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md)
+for more information on the various options available.
+
+You can test the operator by creating a minimal cluster by following this procedure:
+
+1. Create a file called `simple-cluster.yaml` that defines a Redis Enterprise
+   cluster with three nodes:
+
+    ```
+    cat <<EOF > simple-cluster.yaml
+    apiVersion: "app.redislabs.com/v1"
+    kind: "RedisEnterpriseCluster"
+    metadata:
+      name: "test-cluster"
+    spec:
+      nodes: 3
+    EOF
+    ```
+
+    This will request Redis Enterprise nodes with 2 CPUs and 4GB of memory.
+    If you want to test with a larger or smaller configuration, you can
+    specify the node resources. For example, we can reduce the CPU and memory
+    for a quick test:
+
+    ```
+    cat <<EOF > simple-cluster.yaml
+    apiVersion: "app.redislabs.com/v1"
+    kind: "RedisEnterpriseCluster"
+    metadata:
+      name: "test-cluster"
+    spec:
+      nodes: 3
+      redisEnterpriseNodeResources:
+        limits:
+          cpu: 1000m
```

Comment:
I would not run on anything lower than 2 CPU and 3GB in a K8s environment and might be worthwhile to point to HW requirements doc since this isn't a configuration supported, even for a dev cluster.

Reply:
I do this all the time in my testing but I won't use that in an example in these docs. :)